rotkehlchen/db/cache.py

```diff
@@ -36,6 +36,12 @@ class AddressArgType(TypedDict):
     address: ChecksumEvmAddress
 
 
+class ExtraTxArgType(TypedDict):
+    """Type of kwargs, used to get the value of `DBCacheDynamic.EXTRA_INTERNAL_TX`"""
+    tx_hash: str
```

Comment:
Isnt the type `EVMTxHash` here?
```suggestion
    tx_hash: EVMTxHash
```